runtime/compiler/runtime/ClassUnloadAssumption.cpp

```diff
@@ -716,6 +716,11 @@ void TR_UnloadedClassPicSite::compensate(TR_FrontEnd *, bool isSMP, void *)
    value |= 0x03a00001;
    *((uint32_t *)_picLocation) = value;
    armCodeSync(_picLocation, 4);
+#elif defined(TR_HOST_ARM64)
+   // On aarch64, we use constant data snippet for class unloading pic site
```

Comment:
Where does AArch64 do this in a constant data snippet?  I'm looking for the code and can't seem to find it.

Reply:
AArch64 does this here.
https://github.com/eclipse/omr/blob/55f3fb746ea8878d3323ffbfb798ea6155268d9a/compiler/aarch64/codegen/ConstantDataSnippet.cpp#L101-L109

We have not implemented `aconstEvaluator` to use ConstantDataSnippet for class unloading pic, though. Z does it like this:
https://github.com/eclipse/omr/blob/55f3fb746ea8878d3323ffbfb798ea6155268d9a/compiler/z/codegen/OMRCodeGenerator.cpp#L4019-L4023